frontend/require/RetryExamTestSidebarView.html

```diff
@@ -2,6 +2,12 @@
   <div class="panel-heading"><h3 class="panel-title"><a href="#ti/{{tid}}/{{tiNumber}}">{{testShortName}}</a></h3></div>
   <div class="panel-body">
 
+    {{#open}}
+    {{#allQuestionsAnswered}}
+    <p>You have answered every question. Return to Assessment overview to finish the exam.</p>
```

Comment:
I wonder whether this is noticeable enough? We could add a [`text-danger`](http://getbootstrap.com/css/#helper-classes-colors). Or even a [dismissible alert](http://getbootstrap.com/components/#alerts-dismissible) at the top of the page (in the top-level `#error` div?). I'm not sure exactly how visible this needs to be.